skins/oasis/css/core/ads-porvata.scss

```diff
@@ -0,0 +1,282 @@
+		}
+
+		&.vpaid-enabled video {
+			z-index: $vpaid-z-index;
+		}
+
+		.video-player {
+			background: #000;
+			margin: 0 auto;
+			pointer-events: auto;
+		}
+	}
+
+	.replay-overlay {
+		background: rgba(0,0,0,0.3) url('/extensions/wikia/AdEngine/js/video/player/ui/images/play_button.svg') center/8% no-repeat;
+		cursor: pointer;
+		height: 100%;
+		position: absolute;
+		z-index: 1;
+	}
+}
+
+.video-player {
+	position: relative;
+
+	&#{&}-left, &#{&}-right {
+		z-index: $wikia-top-ads-index;
+	}
+
+	&#{&}-right {
+		right: 0;
+	}
+
+	video {
+		left: 0;
+	}
+
+	.progress-bar {
+		background: $default-progress-bar-color;
+		bottom: 0;
+		height: 8px;
+		left: 0;
+		position: absolute;
+		width: 100%;
+	}
+
+	.current-time {
+		@include transition(width, 1s, linear, 0);
+		background: $default-progress-bar-current-time-color;
+		height: 100%;
+		max-width: 100%;
+		width: 0;
+	}
+
+	.pause-overlay {
+		cursor: pointer;
+		height: 100%;
+		position: absolute;
+		top: 0;
+		width: 100%;
+	}
+
+	.close-ad {
+		background-image: url('/extensions/wikia/AdEngine/js/video/player/ui/images/close_button.svg');
+		cursor: pointer;
+		height: $video-player-button-size;
+		margin: 10px 10px 30px 30px;
+		position: absolute;
+		right: 0;
+		top: 0;
+		width: $video-player-button-size;
+	}
+
+	/* TODO remove .ima-mute-div and all references in next releases (keep .volume-control only) */
+	.ima-mute-div, .volume-control {
+		/* remove Firefox's dotted outline on buttons and links */
+		:focus {outline:none;}
+		::-moz-focus-inner {border:0;}
+
+		bottom: 18px;
+		cursor: pointer;
+		position: absolute;
+		right: 10px;
+
+		.speaker {
+			&.mute {
+				background-image: url('/extensions/wikia/AdEngine/js/video/player/ui/images/sound_off_button.svg');
+			}
+
+			background-image: url('/extensions/wikia/AdEngine/js/video/player/ui/images/sound_on_button.svg');
+			display: inline-block;
+			height: $video-player-button-size;
+			overflow: hidden;
+			position: relative;
+			width: $video-player-button-size;
+		}
+	}
+}
+
+#INCONTENT_WRAPPER {
+	font-size: 0;
+	line-height: 0;
+	text-align: center;
+
+	#INCONTENT_PLAYER.porvata-outstream {
+		margin: 0 auto;
+		max-width: 640px;
+
+		&:before {
+			color: $color-text;
+			content: attr(data-label);
+			display: block;
+			font-size: 0.625rem;
+			font-weight: 700;
+			line-height: 0.75rem;
+			padding-bottom: 6px;
+			padding-top: 6px;
+			text-align: center;
+			text-transform: uppercase;
+		}
+
+		.video-overlay {
+			background: #000;
+		}
+	}
+}
+
+/* Outstream */
+
+.porvata-outstream {
+	.video-player {
+		.progress-bar {
+			background: $outstream-progress-bar-color;
+			height: 5px;
+		}
+
+		.current-time {
+			background: $outstream-progress-bar-current-time-color;
+		}
+
+		.ima-mute-div, .volume-control {
+			bottom: 14px;
+
+			.speaker {
+				&.mute {
+					background: #fff;
+					mask-image: url('/extensions/wikia/AdEngine/js/video/player/ui/images/outstream_sound_off_button.svg');
```

Comment:
Can we avoid using `mask-image`? It requires duplicating inlined image and is not supported by IE